src/assets/wallets-logos/LobstrIcon.svelte

```diff
@@ -0,0 +1 @@
+<img style="height: 100%; border-radius: 10px;" src="/lobstr.webp" alt="Lobstr Icon" />
```

Comment:
Can we replace this `.webp` logo for [this](https://static.lobstr.co/static/lobstr/smart_banners/store_icons/icon128.png) one?

Reply:
done